server/views/docs/cms-find-content.njk

```diff
@@ -0,0 +1,60 @@
+{% extends 'docs/cardigan.njk' %}
+
+{% block body %}
+  <p>Content management is being handled through <a href="https://prismic.io">Prismic</a>.</p>
+  <p>
+    We will run through the specific steps to content management for
+    <a href="https://next.wellcomecollection.org">Wellcome Collection</a>
+    but you can find more thorough documentation in the
+    <a href="https://intercom.help/prismicio">Prismic documentation</a>
+  </p>
+
+  <h2>Editorial content</h2>
+
+  <p>To get started creating or editing editorial content, follow the steps below:</p>
+
+  <h3>Login</h3>
+  <p>
+    Login to <a href="https://wellcomecollection.prismic.io">Prismic</a>.
+    If you don't have a login, please email the
```

Comment:
Didn't add the actual email address as this is going to be public.
I assume the docs will be moved from here, or more hopefully deleted.